.github/build-workflows.ps1

```diff
@@ -1,3 +1 @@
-Get-ChildItem -Path './pkl-workflows' -Filter *.pkl -File -Name | ForEach-Object {
-    &pkl eval ./pkl-workflows/$_ -o "./workflows/$($_.Replace('pkl', 'yml'))"
-}
\ No newline at end of file
+pkl eval pkl-workflows/*.pkl -o workflows/%{moduleName}.yml
```

Comment:
I don't know if this is correct.
Please check this if this still works.
I'm not a Windows user...